### PR TITLE
OAK-10730 : log a warn for a MongoException previously swallowed

### PR DESCRIPTION
**Pull request by stefan-egli**

### CODE REVIEW

File: oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java [hunk 1]
```diff
@@ -1752,6 +1752,8 @@ public <T extends Document> boolean create(Collection<T> collection, List<Update
                 }
                 return false;
             } catch (MongoException e) {
+                LOG.warn("Encountered MongoException while inserting documents : {} - exception : {}",
```
Comment (reschke) [inline oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java]:
```suggestion
                LOG.warn("Encountered MongoException while inserting documents: {} - exception: {}",
```

**Review by reschke — COMMENTED** [review]
I'd remove some whitespace.

We probably should also check that these do not happen in ITs (or if they do, when...?)

**Comment by stefan-egli** [comment]
There is code that asserts that `ds.create()` returns false - eg [BasicDocumentStoreTest#testCreatePartialFailure](https://github.com/apache/jackrabbit-oak/blob/f9776cf827a7c4ceed5d13e9dc97e924e9722195/oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/BasicDocumentStoreTest.java#L891)

Regarding ITs : you mean upstream ITs or some in oak itself?

**Comment by reschke** [comment]
ITs: I was referring just to Oak ITs...

**Review by reschke — APPROVED** [review]

**Comment by stefan-egli** [comment]
thx!

**Pull request merged**